2_deploy_ses_aio/play.yml

```diff
@@ -0,0 +1,74 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+- name: Install SES 5
+  hosts: ses_nodes:ses
+  remote_user: root
+  gather_facts: true
+  roles:
+    - ses
+  vars:
+    ses_openstack_config: True
+
+- hosts: ses_nodes[0]
+  vars:
+    ses_config_file_path: "~/suse-osh-deploy/ses_config.yml"
```

Comment:
That could be great if this makes uses of vars/common-vars.yml. I suppose you want to do that in another PR?

On top of that, I haven't decided yet the filename we should have for this, as ansible-runner doesn't plan for that case. Either `env/extra_files/ses_config.yml`, or `artifacts/ses_config.yml`. In the meantime, we can keep it that way, but we should make sure this is tracked as somethimg we do (assign a case in jira).

Reply:
> That could be great if this makes uses of vars/common-vars.yml. I suppose you want to do that in another PR?

Yup, this is only a small logical change so adding a different change under the same PR feels wrong to me

Reply:
OK.